File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.rs
impl Solution {
    pub fn longest_consecutive(mut nums: Vec<i32>) -> i32 {
     if nums.is_empty() {
        return 0;
     }
     nums.sort();
     let mut count =1;
     let mut max_count =1;

     for i in 1..nums.len(){
        if nums[i] == nums[i-1] {
            continue;
        }
        if nums[i] == nums[i-1]+1{
            count +=1;
            
           
        }
        else {
count =1;
        }
        max_count = max_count.max(count);
        
     }
     max_count
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-3.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numbers = []
        for i , num in enumerate(nums):
            numbers.append([num,i])
        numbers.sort()
        i , j = 0, len(numbers) -1
        while i < j:
            curr = numbers[i][0] + numbers[j][0]
            if curr == target:
                return [min(numbers[i][1],numbers[j][1]),max(numbers[i][1],numbers[j][1])]
            elif curr < target:
                i +=1
            else:
                j-=1
        return []
        